public/js/main.js

```diff
@@ -216,12 +216,34 @@
             })
                 .then(function(res) {
                     updateLastOpened();
-                    $scope.messages = res.data;
+                    updateMessages(res.data);
                 }, function(response) {
                 }
             );
         };
 
+        function updateMessages(newM) {
+            if (newM === []) {
```

Comment:
Is this function meant to clear all the current messages if no new messages are received?